common/test/java/com/couchbase/lite/DbCollectionsTest.kt

```diff
@@ -146,7 +146,6 @@ class DbCollectionsTest : BaseCollectionTest() {
 
 // !!! TESTS BELOW NEED TO BE MOVED TO CollectionTest
 
-    @Ignore("CBL-3257: getScopeNames does not return default scope when it is empty")
```

Comment:
Oh!  Cool!  This is the one I just noticed, too.  Might be a merge conflict here.